lib/console.js

```diff
@@ -64,10 +64,7 @@ Console.prototype.time = function(label) {
 
 
 Console.prototype.timeEnd = function(label) {
-  var time = this._times.get(label);
-  if (!time) {
```

Comment:
This check still needs to be here. And if the entry isn't there, you'd need to return.